app/models/gsa18f/procurement_fields.rb

```diff
@@ -18,6 +18,24 @@ def display
       attributes_for_view.push(["Total Price", procurement.total_price])
     end
 
+    def email_display
+      [
+        ["Purchase Type", procurement.purchase_type],
+        ["Date Requested", procurement.date_requested],
+        ["Quantity", procurement.quantity],
+        ["Urgency", procurement.urgency_string],
+        ["Cost Per Unit", procurement.cost_per_unit],
+        ["Office", procurement.office],
+        ["Total Price", procurement.total_price],
+        ["Justification", procurement.justification],
+        ["Link to Product", procurement.link_to_product],
+        ["Additional Info", procurement.additional_info]
+      ]
+    end
```

Comment:
this model is used for picking out which fields belong in views. It's kind of a weird duck. 

Was extracted from some methods that used to be in the `Gsa18f::Procurement` and `Ncr::WorkOrder` models themselves.

Where would you put it?

Agreed on i18n. We are moving all email stuff there right now. Will do that for this.